src/sedtrails/transport_converter/format_converter.py

```diff
@@ -1,80 +1,544 @@
+        }
+        
+        # Create SedtrailsData object
+        sedtrails_data = SedtrailsData(
+            times=seconds_since_ref,
+            reference_date=self.reference_date,
+            x=data['x'],
+            y=data['y'],
+            bed_level=data['bed_level'],
+            depth_avg_flow_velocity=depth_avg_flow_velocity,
+            fractions=1,  # Default to 1 fraction
+            bed_load_sediment=bed_load_sediment,
+            suspended_sediment=suspended_sediment,
+            water_depth=data['water_depth'],
+            mean_bed_shear_stress=data['mean_bed_shear_stress'],
+            max_bed_shear_stress=data['max_bed_shear_stress'],
+            sediment_concentration=data['sediment_concentration'],
+            nonlinear_wave_velocity=nonlinear_wave_velocity
+        )
+        
+        return sedtrails_data
 
 
 if __name__ == "__main__":
-    data = SedtrailsData(   time=0,
-                            x=0,
-                            y=0,
-                            time_step=0,
-                            bed_level=0.0,
-                            depth_avg_flow_velocity=0.0,
-                            fractions=0,
-                            bed_load_sediment=0.0,
-                            suspended_sediment=0.0,
-                            water_depth=0.0,
-                            mean_bed_shear_stress=0.0,
-                            max_bed_shear_stress=0.0,
-                            sediment_concentration=0.0,
-                            nonlinear_wave_velocity=0.0)
-    print(data)
-
-    data.previous = 'previous data'
-    print(data.previous)
-
-    # print(data.data_instance())
+    # Example usage
+    import sys
+    
+    # Default file path (can be overridden by command line argument)
+    file_path = r"c:\Users\weste_bt\GitHub\sedtrails_data\inlet_sedtrails.nc"
+    
+    # Allow command line override of file path
+    if len(sys.argv) > 1:
+        file_path = sys.argv[1]
+    
+    # Create converter and read data
+    # Using default reference date (1970-01-01, Unix epoch)
+    converter = FormatConverter(file_path, input_type=InputType.NETCDF_DFM)
+    converter.read_data()
+    
+    # Print time information
+    time_info = converter.get_time_info()
+    print("Time information:")
+    print(f"  Reference date: {time_info['reference_date']}")
+    print(f"  Start time: {time_info['time_start']}")
+    print(f"  End time: {time_info['time_end']}")
+    print(f"  Number of time steps: {time_info['num_times']}")
+    
+    # Convert all time steps to SedtrailsData
+    sedtrails_data = converter.convert_to_sedtrails_data()
+    
+    # Print some information about the converted data
+    print("\nConverted data:")
+    print(f"  Number of time steps: {len(sedtrails_data)}")
+    print(f"  Reference date: {sedtrails_data.reference_date}")
+    print(f"  Time values range: {sedtrails_data.times[0]} to {sedtrails_data.times[-1]} seconds since reference")
```

Comment:
Can you move these examples to the examples directory? you can create a new python script which import the class and then add the example code. That way we won't forget that we have an example, and we will document it later on